snowfall/decoding/util.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c)  2021  Xiaomi Corp.       (author: Fangjun Kuang)
+
+import k2
+import torch
+
+
+def get_log_probs(phone_fsas: k2.Fsa, nnet_output: torch.Tensor,
```

Comment:
@danpovey 

If I understand correctly, this is how to compute the log-prob for the reference input phone labels. Correct me if I am wrong.

The input `phone_fsas` contains epsilon self-loops and `nnet_output`
is the output of the second pass model after log-softmax.